examples/stable/test/property/ic_api/call/.gitignore

```diff
@@ -0,0 +1,2 @@
+.azle
```

Comment:
This .gitignore file does not look like the others in the `ic_api` directory, it should look exactly like the `crypto_get_random_values` file. No changes should have been made to any file except `tests.ts`.